src/coreclr/vm/crst.h

```diff
@@ -105,6 +105,8 @@ extern DWORD g_fEEShutDown;
 extern Volatile<LONG> g_ShutdownCrstUsageCount;
 extern Volatile<LONG> g_fForbidEnterEE;
 
+class PendingTypeLoadTable;
+
```

Comment:
This shouldn't be necessary. A `friend` declaration in the manner below is sufficient.

Reply:
Got rid of that